src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+$font-color-primary: #060b35;
+$font-color-collateral: #616070;
+$background-color:#fff;
+$border-color: #f3f3f3;
+$button-color:#00acdc;
+$hug: 16px;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $background-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  padding: $hug;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
```

Comment:
1. Instead of `margin-top` here better add `padding: 32px 16px 16px` to `.card`;
2. `margin-left: 3px` will work only for width of 200px, but we what the image to be entered always. (`margin: 0 auto`)